addendum.dict

```diff
@@ -357,6 +374,7 @@ Tartary	𐑑𐑸𐑑𐑼𐑦
 Tavernier	𐑑𐑨𐑝𐑺𐑯𐑦𐑱
 Tay	𐑑𐑱
 Tello	𐑑𐑧𐑤𐑴
+Tewkesbury 𐑗𐑫𐑒𐑕𐑚𐑮𐑰
```

Comment:
I've set out the approach to yod-coalescence in point 9 at https://www.shavian.info/dictionary/. I'm not yet reflecting variant pronunciations in this case (maybe in future), so would suggest removing this,